servatrice/src/servatrice.cpp

```diff
@@ -95,16 +95,16 @@ Servatrice_ConnectionPool *Servatrice_GameServer::findLeastUsedConnectionPool()
 {
     int minClientCount = -1;
     int poolIndex = -1;
-    QStringList debugStr;
+    QVector<int> debugValue;
     for (int i = 0; i < connectionPools.size(); ++i) {
         const int clientCount = connectionPools[i]->getClientCount();
         if ((poolIndex == -1) || (clientCount < minClientCount)) {
             minClientCount = clientCount;
             poolIndex = i;
         }
-        debugStr.append(QString::number(clientCount));
+        debugValue.append(clientCount);
     }
-    qDebug() << "Pool utilisation:" << debugStr;
+    qDebug() << "Pool utilisation:" << debugValue;
```

Comment:
this actually does not look like I expected

Reply:
How do we want to address this then?

Reply:
I'll get to this later, should be something like QString() around it or use .arg